libknet/libknet.h

```diff
@@ -576,6 +577,23 @@ int knet_handle_enable_filter(knet_handle_t knet_h,
 
 int knet_handle_setfwd(knet_handle_t knet_h, unsigned int enabled);
 
+/**
+ * knet_handle_setprio_dscp
+ *
+ * @brief Use dscp for IP_TOS on socket to implement KNET_LINK_FLAG_TRAFFICHIPRIO
+ *
+ * knet_h   - pointer to knet_handle_t
+ *
+ * dscp     - dscp value to set on all new sockets
+ *
+ * @return
+ * knet_handle_setfwd returns
```

Comment:
nice copy paste ;) as above, please expand the doc that it needs to be set before configuring links.